code/mob/living/critter/sawfly.dm

```diff
@@ -189,8 +189,8 @@ This file is the critter itself, and all the custom procs it needs in order to f
 					boutput(user, "You collapse [src].")
 					src.foldself()
 		else
-			if(prob(50))
-				boutput(user,"<span class='alert' In your attempt to pet the [src], you cut yourself on its blades! </span>")
+			if(prob(50)&& isalive(src))
+				visible_message(user,"<span class='alert' In their attempt to pet the [src], [user] cuts themselves! </span>")
```

Comment:
You should be able to use the third parameter to do the second person POV I believe.